conf/bblayers.conf

```diff
@@ -52,4 +52,8 @@ BBLAYERS = " \
   ${BASELAYERS} \
   ${TOPDIR}/sources/openembedded-core/meta \
   ${TOPDIR}/sources/meta-rockchip \
+  /home/kpoduval/workspace/yoe-zcu/sources/meta-xilinx/meta-xilinx-bsp \
+  /home/kpoduval/workspace/yoe-zcu/sources/meta-xilinx/meta-xilinx-contrib \
+  /home/kpoduval/workspace/yoe-zcu/sources/meta-xilinx/meta-xilinx-pynq \
+  /home/kpoduval/workspace/yoe-zcu/sources/meta-xilinx/meta-xilinx-standalone \
```

Comment:
I think its hard-coding paths here perhaps a bug in our layer adding tool but please replace it with `${TOPDIR}`

Reply:
Good catch, it looks like the very next patch in the series fixed it to ${TOPDIR} (looks like yoe_*/bitbake-* scripts did the magic. Should I squash all the xilinx related patches to one ? Also should these be moved to BSPLAYERS instead of BBLAYERS ? If yes then the rockchip layer should be moved too. Please confirm.

Reply:
I think just squash the one which fixes the above issue into one. Rest seem to be ok